sdk/cosmosdb/cosmos/src/ClientContext.ts

```diff
@@ -55,12 +54,16 @@ export class ClientContext {
   private diagnosticWriter: DiagnosticWriter;
   private diagnosticFormatter: DiagnosticFormatter;
   public partitionKeyDefinitionCache: { [containerUrl: string]: any }; // TODO: PartitionKeyDefinitionCache
+  /** boolean flag to support operations with client-side encryption */
+  public enableEncryption: boolean = false;
+
   public constructor(
     private cosmosClientOptions: CosmosClientOptions,
     private globalEndpointManager: GlobalEndpointManager,
     private clientConfig: ClientConfigDiagnostic,
     public diagnosticLevel: CosmosDbDiagnosticLevel,
   ) {
+    this.enableEncryption = cosmosClientOptions.encryptionPolicy?.enableEncryption ? true : false;
```

Comment:
is there an default value of enableEncryption. if yes then we don't need this  ? true : false;

Reply:
updated